tests/unit/certGen.unitTest.ts

```diff
@@ -2307,7 +2307,8 @@ describe("cert-gen", () => {
     context("CertGen function", () => {
         context("when a passing test result is read from the queue", () => {
             context("and the payload generation throws an error", () => {
-                it("should bubble that error up", async () => {
+                // causes a time-out
```

Comment:
I am confused as to why we are skipping this? Are we saying these changes have caused this behaviour?